### ISSUE TITLE
NameError in inlineCallbacks' python3.3+ documentation example

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7763]
|<img alt="mineo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mineo reported|
|-|-|
|Trac ID|trac#7763|
|Type|enhancement|
|Created|2015-01-30 17:03:17Z|
|Branch|https://github.com/twisted/twisted/tree/inlineCallbacks-typo-7763|

The example currently is
```
response = yield makeRequest(url)
return json.loads(respoonse)
```
(note the difference between "response" and "respoonse". The attached patch removes one of "respoonses" o's.

Attachments:

* [inlinecallbacks-example.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d5b/d5bbad7af200e871991f55691eb2c40b41d80868/4bfa753a7416ceb3ac1ea2ac71ae742070ed7b72.patch) (466 bytes) - added by mineo on 2015-01-30 17:03:37Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7763 7763
type__enhancement enhancement
reporter__mineo mineo
priority__normal normal
milestone__ 
branch__branches_inlineCallbacks_typo_7763 branches/inlineCallbacks-typo-7763
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1422637397215384 1422637397215384
changetime__1422862501583544 1422862501583544
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

(In [43800]) Branching to inlineCallbacks-typo-7763.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph set owner to @glyph|
|-|-|

Good catch, mineo! Thanks for contributing this fix. Buildbot looks good. I will commit momentarily.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set status to `closed`|
|-|-|

(In [43803]) Merge inlineCallbacks-typo-7763: Fix a typo in inlineCallbacks's docstring.

Author: mineo

Reviewer: glyph

Fixes: [#7763](https://github.com/twisted/twisted/issues/7763)

The variable should be called "response" not "respoonse".
